spark/src/main/kotlin/com/adevinta/spark/components/textfields/Dropdown.kt

```diff
@@ -313,129 +303,252 @@ public fun SelectTextField(
+ * @param onDismissRequest Called when the user requests to dismiss the menu, such as by
+ * tapping outside the menu's bounds
+ * @param modifier a [Modifier] for this text field
+ * @param enabled True controls the enabled state of the [TextField]. When `false`, the text field will
+ * be neither editable nor focusable, the input of the text field will not be selectable,
+ * visually text field will appear in the disabled UI state
+ * @param required add an asterisk to the label to indicate that this field is required and read it as "label mandatory"
+ * but doesn't do anything else so it's up to the developer to handle the behavior.
+ * @param label the optional label to be displayed inside the text field container. The default
+ * text style for internal [Text] is [SparkTypography.body2] when the text field is in focus and
+ * [SparkTypography.body1] when the text field is not in focus
+ * @param placeholder the optional placeholder to be displayed when the text field is in focus and
+ * the input text is empty. The default text style for internal [Text] is [SparkTypography.body1]
+ * @param helper The optional helper text to be displayed at the bottom outside the text input container that give some
+ * informations about expected text
+ * @param leadingContent the optional leading icon to be displayed at the beginning of the text field
+ * container
+ * @param state indicates the validation state of the text field. The label, outline, leading & trailing content are tinted by the state color.
+ * @param stateMessage the optional state text to be displayed at the helper position that give more information about
+ * the state, it's displayed only when [state] is not null.
+ * @param visualTransformation transforms the visual representation of the input [value]
+ * For example, you can use [PasswordVisualTransformation][androidx.compose.ui.text.input.PasswordVisualTransformation]
+ * to create a password text field. By default no visual transformation is applied
+ * @param properties commentCountPopupProperties for further customization of this popup's behavior.
+ * @param interactionSource the [MutableInteractionSource] representing the stream of
+ * [Interaction]s for this TextField. You can create and pass in your own remembered
+ * [MutableInteractionSource] if you want to observe [Interaction]s and customize the
+ * appearance / behavior of this TextField in different [Interaction]s.
+ * @param dropdownContent The content to be displayed inside ExposedDropdownMenuBox.
  */
+@OptIn(ExperimentalMaterial3Api::class)
 @ExperimentalSparkApi
 @Composable
-public fun SparkSelectTrailingIcon(
+public fun Dropdown(
+    value: String,
     expanded: Boolean,
+    onExpandedChange: (Boolean) -> Unit,
+    onDismissRequest: () -> Unit,
     modifier: Modifier = Modifier,
     enabled: Boolean = true,
-    onIconClick: () -> Unit = {},
+    required: Boolean = false,
```

Comment:
I prefer to use use 
isVerb
to communicate that it's a True/False question

this would be `isEnabled, isExpanded, isRequired`

Reply:
hmmmmm, up to you then, we can discuss it later